include/gsl/span

```diff
@@ -646,52 +678,52 @@ private:
     span<element_type, dynamic_extent> make_subspan(index_type offset, index_type count,
                                                     subspan_selector<dynamic_extent>) const
     {
-        Expects(offset >= 0 && size() - offset >= 0);
+        Expects(size() >= offset);
 
         if (count == dynamic_extent) { return {KnownNotNull{data() + offset}, size() - offset}; }
 
-        Expects(count >= 0 && size() - offset >= count);
+        Expects(size() - offset >= count);
         return {KnownNotNull{data() + offset}, count};
     }
 };
 
 #if defined(GSL_USE_STATIC_CONSTEXPR_WORKAROUND)
-template <class ElementType, std::ptrdiff_t Extent>
+template <class ElementType, std::size_t Extent>
 constexpr const typename span<ElementType, Extent>::index_type span<ElementType, Extent>::extent;
 #endif
 
 // [span.comparison], span comparison operators
```

Comment:
These were removed for C++20 by [P1085R2](https://wg21.link/P1085R2).

Reply:
These will be moved in a different pr.